content/applications/sales/subscriptions/payment_providers/sdd.rst

```diff
@@ -0,0 +1,222 @@
+=================
+SEPA Direct Debit
+=================
+
+.. |sdd| replace:: :abbr:`SDD (SEPA Direct Debit)`
+
+*SEPA*, or Single Euro Payments Area, is a payment-integration initiative of the European Union for
+simplifying bank transfers denominated in euros.
+
+*SEPA Direct Debit (|sdd|)* is a payment provider that allows future payments to be collected from
+customers' bank accounts by signing a mandate. This allows recurring payments to be automatically
+gathered and processed using |sdd|.
+
+.. important::
+   To create direct debit mandates, the **Accounting** app **must** be installed.
+
+.. _subscriptions/sepa-sdd-configuration:
+
+Configuration
+-------------
+
+.. important::
+   Before using |sdd|, the database must be configured to allow the use of Euros on sales orders. To
+   do so, a :doc:`pricelist <../../point_of_sale/pricing/pricelists>` must be created for Euros.
+
+   This can be done using the **Sales** or **Point of Sales** apps. From either app, navigate to
+   :menuselection:`Configuration --> Settings`.
+
+   In the :guilabel:`Pricing` section, underneath the :guilabel:`Pricelists` feature, click
+   :icon:`fa-arrow-right` :guilabel:`Pricelists`. Then, click the :guilabel:`New` button to open a
+   new pricelist form. Add a name for the pricelist in the top field, then set the drop-down menu
+   next to :guilabel:`Currency` to :guilabel:`EUR`.
+
+   .. image:: sdd/euro-pricelist.png
+      :align: center
+      :alt: A pricelist using EUR as the currency.
+
+To use the wire transfer payment method, first ensure the |sdd| payment method is enabled. To do so,
+navigate to :menuselection:`Accounting app --> Configuration --> Settings`, scroll to the
+:guilabel:`Customer Payments` section, and tick the :guilabel:`SEPA Direct Deposit (SDD)` checkbox.
+Then, set a creditor identifier number in the field labeled, :guilabel:`Creditor Identifier`. Then,
+click :guilabel:`Save`.
+
+.. tip::
+   The company using |sdd| must have a creditor identifier to issue a SEPA Direct Debit payment
+   request. If using Test Mode, use the test creditor identifier number: `DE98ZZZ09999999999`, and
+   click :guilabel:`Save`.
```

Comment:
Also suggestion to link to this section, otherwise please remove the capitalization :)
```suggestion
   request. If using :ref:`test mode <subscriptions/sepa-sdd-test-mode>`, use the test creditor
   identifier number: `DE98ZZZ09999999999`, and click :guilabel:`Save`.
```